.circleci/config.yml

```diff
@@ -22,12 +31,21 @@ commands:
       - run: make build
       - run: make release
 
-  common_setup:
-    description: Prepare the repository for work
+  perform_opera_nightly_integration_testing:
+    description: Test xOpera CLI commands with testing service template
     steps:
       - checkout
```

Comment:
Aha thanks for clarifying things here and also for Sensu Go example. I've noticed that opera already has one stable branch called `stable-0.1`.  Could we use that one or if not what would be the next step here and how would this be properly done for opera?

Reply:
I would suggest that we create a new `stable` or `released` branch that always points for the latest released tag. Then we can follow the same procedure as we do in Sensu Go.

Reply:
Ok, so if I udnerstand correctly we would have to maintain that `stable` branch to point to the lastest relased tag, right? So basically we would do a rebase onto master every time? As a replacement for CircleCI code checkout could we just use some run commands here to clone the opera repository and then just checkout to the latest tag somehow without creating a new separate branch?

Reply:
In the git world, the "latest tag" is not well defined because it has no semantic meaning beyond "point to one and exactly one commit". Any other semantic meaning needs to be attached by humans.

The problem with the automatic determination of the latest tag is in the implementation details. Because git tags can have just about any name, the code that finds the latest stable one would need to be quite robust. But I do not mind adding it to the repo if we decide to go that way.

Reply:
I understand. So now I modified CircleCI config that it will create and push a new `stable` branch every time we create a new tag and deploy something to production. This way we won't need to maintain this manually and therefore our stable branch will always point to the latest pushed tag.